.idea/workspace.xml

```diff
@@ -2,10 +2,10 @@
 <project version="4">
```

Comment:
just a general hint: do not commit files under the `.idea` directory (generated by your IDE), especially user-specific settings, like `workspace.xml`. Check this out for more info: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839